__integrations__/index.test.ts

```diff
@@ -430,8 +430,8 @@ describe("Create Message", () => {
           expect(O.isSome(message)).toBeTruthy();
           expect(O.isSome(status)).toBeTruthy();
           expect(O.isSome(content)).toBeFalsy();
-          expect(O.some(status)).not.toHaveProperty("ttl");
-          expect(O.some(message)).not.toHaveProperty("ttl");
+          expect(O.getOrElse(undefined)(status)).not.toHaveProperty("ttl");
+          expect(O.getOrElse(undefined)(message)).not.toHaveProperty("ttl");
```

Comment:
`getOrElse` needs a Lazy function
```suggestion
          expect(O.getOrElseW(() => undefined)(status)).not.toHaveProperty("ttl");
          expect(O.getOrElseW(() => undefined)(message)).not.toHaveProperty("ttl");
```

Reply:
You are right, mb